src/App.jsx

```diff
@@ -42,7 +41,6 @@ export default function App() {
 
   function handleScroll() {
     const cardListElement = document.getElementById("cardList");
-    // if at the bottom of the page && the current page isn't the last page && results aren't loading
```

Comment:
"if at the bottom of the page" explains `cardListElement.clientHeight + cardListElement.scrollTop === cardListElement.scrollHeight`